api/oapigen/types.go

```diff
@@ -22,6 +29,14 @@ type CatalogServicesCondition_NodeMeta struct {
 	AdditionalProperties map[string]string `json:"-"`
 }
 
+// Condition defines model for Condition.
+type Condition struct {
+	CatalogServices *CatalogServicesCondition `json:"catalog-services,omitempty"`
+	ConsulKv        *ConsulKVCondition        `json:"consul-kv,omitempty"`
```

Comment:
```suggestion
	ConsulKV        *ConsulKVCondition        `json:"consul-kv,omitempty"`
```

Can you update the other occurrences to have `Kv` meet go idiomatic casing for initials?

Reply:
Unfortunately the code generation isn't smart enough to figure out abbreviations like KV and ID, so we get `Id` and `Kv`.

I'll look into it some more, but nothing has come up so far in my research for a workaround

Reply:
Ohh this is auto-generated, this is fine as-is then! sorry i'm not very familiar with the code gen. Thanks for the context!

Reply:
All good! I mean, it looks surprisingly authentic for auto-generated code, so I don't blame you